lvm_read.py

```diff
@@ -42,6 +42,21 @@ def _lvm_dump(lvm_data, filename, protocol=-1):
     pickle.dump(lvm_data, output, protocol=protocol)
     output.close()
 
+def get_separator(file):
+    separators = {'Tab': '\t',
+                  'Comma': ','
+    }
+    for line in file:
+        if line.startswith('Separator'):
+            separator = line.strip()[10:]
```

Comment:
This could cause an Indexation error if a writer decides to write a header like:
`Separator;`